### PR TITLE
letting the user finish input before field adjust - so that decimal i…

### PR DESCRIPTION
**Pull request by Nauna**
…s read

### CODE REVIEW

File: src/foam/u2/Element.js [hunk 1]
```diff
@@ -2525,7 +2525,7 @@ foam.CLASS({
   requires: [ 'foam.u2.view.CurrencyView' ],
   properties: [
     [ 'displayWidth', 15 ],
-    [ 'view', { class: 'foam.u2.view.CurrencyView' } ]
+    [ 'view', { class: 'foam.u2.view.CurrencyView', onKey: false } ]
```
Comment (kgrgreer) [inline src/foam/u2/Element.js]: I would just change it in CurrencyView itself.

File: src/foam/u2/tag/Input.js [hunk 2]
```diff
@@ -46,8 +46,9 @@ foam.CLASS({
     {
       class: 'Boolean',
       name: 'onKey',
+      documentation: 'When true, $$DOC{ref:".data"} is updated on every keystroke, rather than on blur.',
       attribute: true,
-      // documentation: 'When true, $$DOC{ref:".data"} is updated on every keystroke, rather than on blur.'
+      value: true
```
Comment (kgrgreer) [inline src/foam/u2/tag/Input.js]: Why is this being set to true?

File: src/foam/u2/tag/Input.js [hunk 3]
```diff
@@ -137,8 +137,6 @@ foam.CLASS({
 
     function fromProperty(p) {
       this.SUPER(p);
-
-      if ( ! this.hasOwnProperty('onKey') && p.validateObj ) this.onKey = true;
```
Comment (kgrgreer) [inline src/foam/u2/tag/Input.js]: I don't think it is safe to remove this.

Comment (Nauna) [reply src/foam/u2/tag/Input.js]: We can't override the property onKey if we don't. The only thing that will work, is changing the call like for UnitValue in Elements.js

...
[ 'view', { class: 'foam.u2.view.CurrencyView' , onKey: false } ]

**Comment by kgrgreer** [comment]
Let's discuss later.

**Pull request closed without merging**